lmfit/parameter.py

```diff
@@ -572,8 +573,13 @@ def create_uvars(self, covar=None):
                     par.stderr = sqrt(covar[vindex, vindex])
             stderr = getattr(par, 'stderr', 0.0)
             if stderr is None:
-                stderr = 0.0
-            uvars[par.name] = ufloat(par.value, stderr)
+                stderr = 0.00
+            if stderr < 1.e-13*abs(par.value):
```

Comment:
`par.value == 0` might be an edge case to handle? I wonder if you could just always filter the warnings here without testing stderr or only test for `stderr==0` instead of with the inequality?

Reply:
Thanks, that's a good suggestion.  I also see that `tiny` is defined here (as 1.e-15).  
I will change to `if stderr < tiny*max(tiny, abs(par.value)):`, just to be super-paranoid about floating point.